cypress/testcases/rancher/rancher_integration.spec.ts

```diff
@@ -102,17 +102,18 @@ describe('Rancher Integration Test', function () {
 
     });
 
-    it('Prepare Harvester VLAN network', () => {
+    it.skip('Prepare Harvester VLAN network', () => {
         cy.login();
 
         network.createVLAN('vlan1', 'default', '1', 'mgmt')
 
     });
 
     it('Rancher import Harvester', { baseUrl: constants.rancherUrl }, () => {
-        // cy.login();
         cy.visit('/');
 
+        rancher.firstTimeLogin()
```

Comment:
https://github.com/harvester/tests/blob/842d76f9d6f55932ad6db33ebd75c1b8aac3a715/cypress/pageobjects/rancher.po.ts#L156 Why does the `firstTimeLogin` judgment need to be added here, maybe repeated here